pb.h

```diff
@@ -344,6 +359,10 @@ struct pb_msgdesc_s {
 
     bool (*field_callback)(pb_istream_t *istream, pb_ostream_t *ostream, const pb_field_iter_t *field);
 
+    bool (*init)(pb_decode_if_t *, void *, unsigned int);
+    bool (*decode)(pb_decode_if_t *, pb_istream_t *, void *, unsigned int, uint32_t, pb_wire_type_t);
+    void (*release)(pb_decode_if_t *, void *, unsigned int, uint32_t);
+
```

Comment:
Could these be put into `PB_ENABLE_DECODE_FAST` ifdef? That would save 12 bytes of flash per message when the feature is not used.